app/helpers/api/bapi/getCatalogueItem.js

```diff
@@ -0,0 +1,11 @@
+import { getData } from 'buying-catalogue-library';
+import { logger } from '../../../logger';
+import { getEndpoint } from '../../../endpoints';
+
+export const getCatalogueItem = async ({ itemId, accessToken }) => {
+  const endpoint = getEndpoint({ api: 'bapi', endpointLocator: 'getCatalogueItem', options: { itemId } });
+  const catalogueItem = await getData({ endpoint, accessToken, logger });
+  logger.info(`Retrieved catalogue item from BAPI for ${itemId}`);
```

Comment:
Just wondering how do you know here if you actually did manage to retrieve a catalogue item? Would it throw an error & therefore wouldn't reach the logger.info line?

Reply:
yeah that's right. an exception would be thrown and that will bubble up to our errorhandler